packages/template-retail-react-app/app/contexts/store-locator-provider.jsx

```diff
@@ -49,7 +51,11 @@ export const StoreLocatorProvider = ({config, children}) => {
 
     const value = {
         state,
-        setState
+        setState,
+        // Modal actions
+        isOpen,
+        onOpen,
+        onClose
```

Comment:
One suggestion, now that we are centeralizing a lot of store locator state in this provider, any chance we can start moving the form control into this provider so we can consistently show the form values for the store locator form?